compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/Scrollable.kt

```diff
@@ -283,13 +283,39 @@ private fun Modifier.pointerScrollable(
             },
             canDrag = { down -> down.type != PointerType.Mouse }
         ))
-        .then(MouseWheelScrollableElement(scrollLogic, scrollConfig, density))
+        .then(MouseWheelScrollableElement(scrollLogicValue, scrollConfig, density))
         .nestedScroll(nestedScrollConnection, nestedScrollDispatcher.value)
 }
 
 // {} isn't being memoized for us, so extract this to make sure we compare equally on recomposition.
 private val NoOpOnDragStarted: suspend CoroutineScope.(startedPosition: Offset) -> Unit = {}
 
+@Composable
+private fun rememberScrollingLogic(
```

Comment:
Remember ScrollingLogic

It seems there is test that checks equality of the modifier across compositions. Probably for optimization.

Remembering scrolling logic seems logical here, and shouldn't add overhead.

In 1.6 it was fully rewritten to modifier node:

https://github.com/androidx/androidx/blob/a7e6bdf524be186baca08e9ddf7272b834a14bd2/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/Scrollable.kt#L276

Reply:
[This one](https://teamcity.jetbrains.com/buildConfiguration/JetBrainsPublicProjects_Compose_DesktopTestsForPullRequests/4287964?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandPull+Request+Details=true&expandBuildTestsSection=true)